hedera-node/hedera-mono-service/src/test/java/com/hedera/test/serde/SerializedForms.java

```diff
@@ -152,7 +152,7 @@ private static <T extends FastCopyable> void assertSameCopySerialization(
     }
 
     private static void generateSerializedData() {
-        GENERATOR_MAPPING.get(SerializableSemVers.class).run();
+        GENERATOR_MAPPING.get(MerkleToken.class).run();
```

Comment:
I don't think it's a big deal that this line changed, but want to check?

Reply:
As I understand it doesn't matter. @tinker-michaelj this is just generate the data and it is fine